guide/private-key-management/introduction.md

```diff
@@ -2,9 +2,54 @@
 layout: guide
 title: Private key management
 nav_order: 2
-permalink: /private-key-management/introduction/
+has_children: true
+permalink: /guide/private-key-management/introduction/
+main_classes: -no-top-padding
+image: /assets/images/guide/private-key-management/page-private-key-management.jpg
 ---
 
+<!--
+
+Editor's notes
+
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/private-key-management/private-key-management.jpg"
+   retina = "/assets/images/guide/private-key-management/private-key-management@2x.jpg"
+   mobile = "/assets/images/guide/private-key-management/private-key-management-mobile.jpg"
+   mobileRetina = "/assets/images/guide/private-key-management/private-key-management-mobile@2x.jpg"
+   alt-text = "Private key management chapter illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
 # Private key management
 
-This chapter is in development at [PR #48](https://github.com/BitcoinDesign/Guide/pull/48){:target="_blank"}. Check the [preview](https://deploy-preview-48--sad-borg-390916.netlify.app/guide/private-key-management/introduction/){:target="_blank"}.  
+When designing a bitcoin-application, the way you choose to manage the [private key]({{ '/guide/glossary/#private-key' | relative_url }}){:target="_blank"} often dictates the fundamental user experience for your users. Being in control of a private key means you have access to the funds it holds. Keeping that key safe is like being in charge of the security of your own bank. It comes with a lot of freedom, but also a big responsibility that many users are not used to. 
```

Comment:
I don't think you need hyphens in "use-case", "wallet-application", "best-practices", and "recovery-phrase".

Reply:
Combed through to de-hyphenate these phrases on all pages